Cargo.toml

```diff
@@ -30,9 +30,15 @@ objc = "0.2.3"
 
 [target.'cfg(target_os = "macos")'.dependencies]
 objc = "0.2.3"
-cocoa = "0.17"
 core-foundation = "0.6"
-core-graphics = "0.16"
+
+[target.'cfg(target_os = "macos")'.dependencies.core-graphics]
+git = "https://github.com/pcwalton/rust-core-foundation"
+branch = "new-appkit-versions"
+
+[target.'cfg(target_os = "macos")'.dependencies.cocoa]
+git = "https://github.com/pcwalton/rust-core-foundation"
+branch = "new-appkit-versions"
```

Comment:
Just for others' reference, it looks like the PR that these git branches are pending on is [here](https://github.com/servo/core-foundation-rs/pull/266).